src/linematch.c

```diff
@@ -0,0 +1,406 @@
+#include <assert.h>
```

Comment:
Can you add the following header here?

/* vi:set ts=8 sts=4 sw=4 noet:
 *
 * VIM - Vi IMproved	by Bram Moolenaar
 *
 * Do ":help uganda"  in Vim to read copying and usage conditions.
 * Do ":help credits" in Vim to see a list of people who contributed.
 * See README.txt for an overview of the Vim source code.
 */

Reply:
added